iris-location-service/src/main/java/iris/location_service/utils/ValidationHelper.java

```diff
@@ -18,4 +23,87 @@ public static boolean isValidAndNotNullPhoneNumber(String phoneNumber) {
 			return false;
 		return phoneNumber.matches(regexPhone);
 	}
+
+	public static boolean isPossibleAttackForRequiredValue(String input, String message) {
+		if (input == null || input.length() <= 0) {
+			log.warn(ErrorMessageHelper.INVALID_INPUT_EXCEPTION_MESSAGE + message);
+			return true;
+		}
+
+		return isPossibleAttack(input, message);
+	}
+
+	public static boolean isPossibleAttack(String input, String message) {
+
+		String[] forbiddenSymbolsArray = {
+			"=",
+			"<",
+			">",
+			"!",
+			"\"",
+			"§",
+			"$",
+			"%",
+			"&",
+			"/",
+			"(",
+			")",
+			"?",
+			"´",
+			"`",
+			"¿",
+			"≠",
+			"¯",
+			"}",
+			"·",
+			"{",
+			"˜",
+			"\\",
+			"]",
+			"^",
+			"ﬁ",
+			"[",
+			"¢",
+			"¶",
+			"“",
+			"¡",
+			"¬",
+			"”",
+			"#",
+			"£",
+			"+",
+			"*",
+			"±",
+			"",
+			"‘",
+			"’",
+			"'",
+			"-",
+			"_",
+			".",
+			":",
+			"…",
+			"÷",
+			"∞",
+			";",
+			"˛",
+			"æ",
+			"Æ",
+			"œ",
+			"Œ",
+			"@",
+			"•",
+			"°",
+			"„" };
+
+		if (input.contains("<script")
+			|| input.contains("SELECT") && input.contains("FROM")
+			|| StringUtils.startsWithAny(input, forbiddenSymbolsArray)) {
```

Comment:
To prevent a potential NPE, the following check should be placed before this:
```
if (input == null)
  return false;
```
---
The upper and lower case of the keywords should not matter. Therefore I would use the following:
```
var inputUpper = input.toUpperCase();
if (inputUpper.contains("<SCRIPT")
			|| inputUpper.contains("SELECT") && inputUpper.contains("FROM")
```

Reply:
I have implemented this.